classes/01class/exercises/c01-aws01/roperto/README.md

```diff
@@ -0,0 +1,87 @@
+# C01-AWS01
+
+## Commands Execution Output
+
+- Commands to create the first EC2 instance and any additional resource required:
+```shell script
+# Create and add private key to agent
+KEY_NAME="devops-test-key"
+aws ec2 create-key-pair --key-name "$KEY_NAME" --query 'KeyMaterial' --output text > ~/.ssh/${KEY_NAME}.pem
+chmod 600 ~/.ssh/devops-test-key.pem
+ssh-add ~/.ssh/devops-test-key.pem
+
+# Find the latest Amazon Linux 2 AMI 
+AMI=$(aws ec2 describe-images --owners amazon \
+        --filters "Name=name,Values=amzn2-ami-hvm-*-x86_64-gp2" \
+        --query "sort_by(Images, &CreationDate)[-1].ImageId" --output text)
+echo "AMI: ${AMI}"
```

Comment:
That's a reference on how `terraform` will create EC2's later. I won't update the video now but I will leave it for when/if I do.